docs/enhancements/template.md

```diff
@@ -26,6 +26,16 @@ To get started with this template:
 The `Metadata` section above is intended to support the creation of tooling
 around the enhancement process.
 
+Make sure that the enhancement covers the following scenarios:
+- REST API and Kubernetes API
+- Late-binding and bind-on-discovery
+- SNO and multinode
+- Day1 (cluster installation) and day2 (adding hosts to an installed cluster)
```

Comment:
```suggestion
- Day1 (cluster installation) and day2 (adding hosts to an installed cluster)
- Day2 both to imported clusters and transformed clusters
```

Reply:
+1 
The comment about both Day2 scenarios is important. We just recently got [MGMT-13461](https://issues.redhat.com/browse/MGMT-13461) that blocked a user from adding hosts with Tang

Reply:
Agreed, added